packages/services/api/src/modules/organization/providers/organization-access-tokens-cache.ts

```diff
@@ -5,11 +5,29 @@ import { Inject, Injectable, Scope } from 'graphql-modules';
 import type Redis from 'ioredis';
 import type { DatabasePool } from 'slonik';
 import { prometheusPlugin } from '@bentocache/plugin-prometheus';
+import { AuthorizationPolicyStatement } from '../../auth/lib/authz';
 import { Logger } from '../../shared/providers/logger';
 import { PG_POOL_CONFIG } from '../../shared/providers/pg-pool';
 import { PrometheusConfig } from '../../shared/providers/prometheus-config';
 import { REDIS_INSTANCE } from '../../shared/providers/redis';
-import { findById, type OrganizationAccessToken } from './organization-access-tokens';
+import {
+  findById,
+  OrganizationAccessTokens,
+  type OrganizationAccessToken,
+} from './organization-access-tokens';
+import { OrganizationMembers } from './organization-members';
+
+export type CachedAccessToken = {
+  id: string;
+  organizationId: string;
+  /** undefined, since older cache records might not contain this property. */
+  projectId?: string | null;
+  /** undefined, since older cache records might not contain this property. */
+  userId?: string | null;
+  authorizationPolicyStatements: Array<AuthorizationPolicyStatement>;
+  hash: string;
+  firstCharacters: string;
+};
```

Comment:
We are now only storing a subset of the access token record in the cache, it did not make sense to also store the "raw" permissions and resources, even though in reality it would probably not make a big performance difference.